### PR TITLE
New version: Insomnia.Insomnia version core@2021.7.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37329)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This seems to be Insomnia Core instead of Insomnia Designer.

**Comment by Trenly** [comment]
> This seems to be Insomnia Core instead of Insomnia Designer.

Insomnia.Insomnia is Insomnia Core?

**Comment by ItzLevvie** [comment]
> > This seems to be Insomnia Core instead of Insomnia Designer.
> 
> Insomnia.Insomnia is Insomnia Core?

Oops, it was for https://github.com/microsoft/winget-pkgs/pull/37330, not this.

For this PR, the portable version is being used instead of the installer.

File: manifests/i/Insomnia/Insomnia/core@2021.7.2/Insomnia.Insomnia.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Kong/insomnia/releases/download/core%402021.7.2/Insomnia.Core-2021.7.2-portable.exe
```
Comment (ItzLevvie) [inline manifests/i/Insomnia/Insomnia/core@2021.7.2/Insomnia.Insomnia.installer.yaml:19]:
```suggestion
  InstallerUrl: https://github.com/Kong/insomnia/releases/download/core%402021.7.2/Insomnia.Core-2021.7.2.exe
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**